### PR TITLE
Make imdb fields visible

### PR DESCRIPTION
**Pull request by EmiEnache**

### CODE REVIEW

File: atlas-api/src/main/java/org/atlasapi/generation/generated/model/ItemInfo.java [hunk 1]
```diff
@@ -621,6 +621,16 @@
                             .withJsonType(JsonType.OBJECT)
                             .build()
             )
+            .add(
+                    FieldInfo.builder()
+                            .withName("year")
```
Comment (tristanbeardmore-gray) [inline atlas-api/src/main/java/org/atlasapi/generation/generated/model/ItemInfo.java]: Was this meant to be duration?

File: atlas-api/src/main/java/org/atlasapi/query/QueryWebModule.java [hunk 2]
```diff
@@ -1140,7 +1142,7 @@ ResourceAnnotationIndex organisationAnnotationIndex() {
                         )
                 )
                 .register(
-                        CONTENT_DETAIL,
+                        CONTENT_DETAIL,     //TODO doesn't work?
```
Comment (tristanbeardmore-gray) [inline atlas-api/src/main/java/org/atlasapi/query/QueryWebModule.java]: TODO?

File: atlas-cassandra/src/main/java/org/atlasapi/content/v2/model/udt/LocalizedTitle.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
+package org.atlasapi.content.v2.model.udt;
+
+import java.util.Objects;
+
+import com.datastax.driver.mapping.annotations.Field;
+import com.datastax.driver.mapping.annotations.UDT;
+
+@UDT(name = "localizedTitle")
```
Comment (obroomhall) [inline atlas-cassandra/src/main/java/org/atlasapi/content/v2/model/udt/LocalizedTitle.java]: is snake_case not the convention?

File: atlas-cassandra/src/main/java/org/atlasapi/content/v2/model/udt/Rating.java [hunk 4]
```diff
@@ -11,6 +11,7 @@
     @Field(name = "value") private float value;
     @Field(name = "type") private String type;
     @Field(name = "publisher") private String publisher;
+    @Field(name = "numberOfVotes") private Long numberOfVotes;
```
Comment (obroomhall) [inline atlas-cassandra/src/main/java/org/atlasapi/content/v2/model/udt/Rating.java]: snake_case again?

File: atlas-core/src/main/java/org/atlasapi/content/Localized.java [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+    private Locale locale;
+
+    /**Constructs & sets the Locale of this object based on a language code, a region code, or both.
+     *
+     * @param languageCode  - preferably a 2 character language code (ISO 639).
+     * @param regionCode    - preferably a 2 character region code (ISO 3166).
+     *
+     * @see java.util.Locale for more info on the codes.
+     */
+    public void setLocale(@Nullable String languageCode, @Nullable String regionCode) {
```
Comment (tristanbeardmore-gray) [inline atlas-core/src/main/java/org/atlasapi/content/Localized.java]: Similar comments for this class as mentioned in Owl changes

File: atlas-legacy/src/main/java/org/atlasapi/system/legacy/DescribedLegacyResourceTransformer.java [hunk 6]
```diff
@@ -66,6 +69,7 @@ public final T apply(F input) {
         }
         described.setThumbnail(input.getThumbnail());
         described.setTitle(input.getTitle());
+//        Locale locale = new Locale();
```
Comment (tristanbeardmore-gray) [inline atlas-legacy/src/main/java/org/atlasapi/system/legacy/DescribedLegacyResourceTransformer.java]: ?

File: atlas-api/src/main/java/org/atlasapi/query/QueryWebModule.java (hunk 2 above)
Comment (EmiEnache) [reply atlas-api/src/main/java/org/atlasapi/query/QueryWebModule.java]: It doesn't work, and it's too out of the scope of this ticket to fix it. I'll add a JIRA ticket to fix it though

File: atlas-cassandra/src/main/java/org/atlasapi/content/v2/model/udt/LocalizedTitle.java (hunk 3 above)
Comment (EmiEnache) [reply atlas-cassandra/src/main/java/org/atlasapi/content/v2/model/udt/LocalizedTitle.java]: No - but it turns out convention is all lowercase in a single word!

File: atlas-cassandra/src/main/java/org/atlasapi/content/v2/model/udt/Rating.java (hunk 4 above)
Comment (EmiEnache) [reply atlas-cassandra/src/main/java/org/atlasapi/content/v2/model/udt/Rating.java]: Good spot

File: atlas-legacy/src/main/java/org/atlasapi/system/legacy/DescribedLegacyResourceTransformer.java (hunk 6 above)
Comment (EmiEnache) [reply atlas-legacy/src/main/java/org/atlasapi/system/legacy/DescribedLegacyResourceTransformer.java]: ?!

**Pull request merged**